app/Controller/CoEnrollmentFlowsController.php

```diff
@@ -159,6 +159,7 @@ function beforeRender() {
       $args['conditions']['status'] = SuspendableStatusEnum::Active;
       $args['conditions']['context'] = array(
         MessageTemplateEnum::EnrollmentApproval,
+        MessageTemplateEnum::ApproverNotification,
```

Comment:
Do we need a new context for this? It's still an Approval related email, we could probably just use `EnrollmentApproval`. If we do want a new context, it should follow the existing naming pattern, eg `EnrollmentApprover`.

Reply:
I renamed the enum variable from ApproverNotification to EnrollmentApprover. The Approver template has its own foreign key. As a result i followed the pattern already applied for the other types of Templates. As far as i am concerned and it is more clear to keep them separate.